src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FunctionComponent } from "react";
+
+import "lbh-frontend/lbh/components/lbh-error-message/_error-message.scss";
+
+type Props = {
+  text?: string;
+  html?: string;
+  id?: string;
+  classes?: string[];
+  attributes?: string;
+  visuallyHiddenText?: string;
+};
+
+const ErrorMessage: FunctionComponent<Props> = props => {
+  return (
+    <span
+      id={props.id}
+      className={`govuk-error-message ${props.classes} lbh-error-message`}
+    >
+      {props.visuallyHiddenText && (
+        <span className="govuk-visually-hidden">
+          {props.visuallyHiddenText}
+        </span>
+      )}
+      {props.html ? props.html : props.text}
```

Comment:
This won't work (React will escape the HTML). We should never pass raw HTML in as props. If you want child nodes, pass in a `children` prop instead.